### PR TITLE
[doc] fix path of some doc as code structure refactor

### PR DESCRIPTION
**Pull request by YanjieGao**

### CODE REVIEW

File: docs/grafana/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ grafana-port: 3000        # port for grafana
 
 # Deployment
 
-The deployment of web portal goes with the bootstrapping process of the whole PAI cluster, which is described in detail in [Tutorial: Booting up the cluster](https://github.com/Microsoft/pai/blob/master/pai-management/doc/cluster-bootup.md).
+The deployment of web portal goes with the bootstrapping process of the whole PAI cluster, which is described in detail in [Tutorial: Booting up the cluster](https://github.com/Microsoft/pai/blob/master/docs/pai-management/doc/cluster-bootup.md).
```
Comment (hao1939) [inline docs/grafana/README.md]: What about using relative path? So it won't break on different branch.

Comment (YanjieGao) [reply docs/grafana/README.md]: fixed to relative

File: docs/pylon/README.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ And before starting nginx, the `nginx.conf` file must be generated using `python
 
 ### For deploying with PAI
 
-The deployment of Pylon goes with the bootstrapping process of the whole PAI cluster, which is described in detail in [Tutorial: Booting up the cluster](https://github.com/Microsoft/pai/blob/master/pai-management/doc/cluster-bootup.md). To configure Pylon, change the following field(s) in the `pylon` section in [services-configuration.yaml](../../examples/cluster-configuration/services-configuration.yaml) file:
+The deployment of Pylon goes with the bootstrapping process of the whole PAI cluster, which is described in detail in [Tutorial: Booting up the cluster](https://github.com/Microsoft/pai/blob/master/docs/pai-management/doc/cluster-bootup.md). To configure Pylon, change the following field(s) in the `pylon` section in [services-configuration.yaml](../../examples/cluster-configuration/services-configuration.yaml) file:
```
Comment (hao1939) [inline docs/pylon/README.md]: And here.

Comment (YanjieGao) [reply docs/pylon/README.md]: fixed

**Pull request merged**